packages/base-driver/lib/basedriver/driver.ts

```diff
@@ -171,6 +175,17 @@ export class BaseDriver<
     return res;
   }
 
+  protected clarifyCommandName(cmd: string, args: string[]): string {
```

Comment:
just a small nits:
Wondered if we could move this method into `lib/helpers/extension-command-name.ts` (or resolveExecuteExtensionName can include this `cmd` check part also) as this has no `this` usage

Reply:
My earlier commit did have that. But the goal is to have this method here to allow the implementing drivers to override it if required from what I understand.

Reply:
Got it. I didn't notice that the intentionally allow subclasses to override it